data-raw/process_data_sets.R

```diff
@@ -272,3 +272,14 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+#-------------------------------------------------------------------------------
+# Babies: Documented in R/babies.R
+#-------------------------------------------------------------------------------
+# Population of 1236 babies from
+# https://wjhopper.github.io/SDS-201/data/babies.csv
+babies <- "https://wjhopper.github.io/SDS-201/data/babies.csv" %>%
+  read_csv() %>%
+  mutate(birthday = date) %>% # Creating new date variable
+  mutate(birthday = as.Date(date, origin = "1958-01-01"))
+
```

Comment:
You need to use `use_data()` to save the data-set as an `.rda` file. Look at examples above!